torchft/local_sgd.py

```diff
@@ -293,6 +307,21 @@ def _set_grads(self) -> None:
                 # No longer needed
                 del self._grads[name]
 
+    def _clear_local_parameters(self) -> None:
+        """
+        Clears the saved copy of the model's parameters
+        """
+        self._local_parameters = {}
+
+    def _merge_parameters(self) -> None:
+        """
+        Merges the local and global parameters.
+        """
+        for name, p in self._model_fragment.named_parameters():
+            torch.lerp(
```

Comment:
this isn't inplace right? having some numerics tests would be nice to catch issues like this